renovate.json

```diff
@@ -0,0 +1,15 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "extends": [
+        "config:recommended",
+    
+        ":pinVersions",
+        ":automergeMinor",
```

Comment:
Depending on if you want any automerges (after passing tests), we could remove of keep this. IMO it's very nice to have, especially if you don't have automatic prod releases

Reply:
As of now, renovate will automerge path and minor version updates, and no matter if all pipelines pass, will _not_ automerge major version updates.